gazebo_plugins/test/worlds/wheelslip_worlds/gazebo_ros_wheelslip_1.world

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" ?>
+<!--
+  Gazebo ROS wheel slip plugin demo
+  Try for example:
+  To change slip compliance of a group of wheels:
+    ros2 param set /trisphere_cycle_slip0/wheel_slip_rear slip_compliance_unitless_longitudinal 1.0
+  To get the current slip compliance of a group of wheels:
+    ros2 param get /trisphere_cycle_slip0/wheel_slip_rear slip_compliance_unitless_longitudinal
+-->
```

Comment:
I think we can remove this comment since this world file isn't really meant to be a demo that people run.

Reply:
Removed the comment in all world files